### PR TITLE
Feature/skeleton feature

### PR DESCRIPTION
**Pull request by AmyJenSandoval**
Create skeleton code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,76 @@
-# alpaca-albatross-practicum-team1
+# Getting Started with Create React App
```
Comment (Leigh-K) [inline README.md]: This should be renamed to
`Getting Started with Volunteer app`

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (Leigh-K) [inline public/index.html]: we can now remove this

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
```
Comment (Leigh-K) [inline public/index.html]: lets kill this too

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (Leigh-K) [inline public/index.html]: we killed this too

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (Leigh-K) [inline public/index.html]: update to our app name

File: src/App.test.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```
Comment (Leigh-K) [inline src/App.test.js]: this will fail, maybe for now change to getting the text from the header, volunteer app

**Review by Leigh-K — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AmyJenSandoval) [reply README.md]: Done

File: public/index.html (hunk 2 above)
Comment (AmyJenSandoval) [reply public/index.html]: Done

File: public/index.html (hunk 3 above)
Comment (AmyJenSandoval) [reply public/index.html]: Done

File: public/index.html (hunk 4 above)
Comment (AmyJenSandoval) [reply public/index.html]: Done

File: public/index.html (hunk 5 above)
Comment (AmyJenSandoval) [reply public/index.html]: Done

File: src/App.test.js (hunk 6 above)
Comment (AmyJenSandoval) [reply src/App.test.js]: Done

**Review by Leigh-K — APPROVED** [review]

**Pull request merged**